dali/pipeline/operators/decoder/nvjpeg_decoder.cu

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#ifdef DALI_USE_NVJPEG
```

Comment:
Decide whether to build the nvJPEG op via cmake instead of `#ifdef`

Reply:
Do you mean exclude nvjpeg_decoder.cu in CMakeLists? 
As this source file acts as a wrapper of nvJPEG for DALI,  I choose to "comment out" this whole source to disable nvJPEG. Maybe doing it via cmake will make CMakeLists.txt a bit more complex.
@JanuszL what do you think?

Reply:
Yes. We already use such a method (see LMDB, proto3, etc). It keeps the source files cleaner at the expense of a little more complexity in the CMakeLists.txt

It also means that if you disable nvJPEG then there will be absolutely nothing nvJPEG-related in the build, no files, nothing.

Reply:
Thank you for your reviewing too.
I will push the commit after finishing lint

Reply:
Ok, I will do it